assets/js/stats.js

```diff
@@ -0,0 +1,21 @@
+const $ = require('jquery');
+global.$ = global.jQuery = $;
+
+import '../css/stats.css';
+
+import 'bootstrap';
```

Comment:
I'm just following what was said about bootstrap, you add the stuff you need, regardless if the parent already did it. Because you may not know what was included, and if it changes you know at least that this file needed bootstrap.

Reply:
Yes, webpack will include it only once, but we have created a custom file bootstrap.scss which includes bootstrap and our custom colors, etc which is included in both templates (GRP and GRIIDC). I don't think we need to include it again to show that this page uses it, its redundant